syncer/syncer_test.go

```diff
@@ -1074,53 +1084,63 @@ func (s *testSyncerSuite) TestRun(c *C) {
 
 	go syncer.Process(ctx, resultCh)
 
-	testCases1 := []struct {
-		sql      string
-		tp       opType
-		sqlInJob string
-		arg      interface{}
-	}{
+	sqls1 := []string{
+		"create database if not exists test_1",
+		"create table if not exists test_1.t_1(id int primary key, name varchar(24))",
+		"create table if not exists test_1.t_2(id int primary key, name varchar(24))",
```

Comment:
what does need another table  to test ?

Reply:
because I add `RouteRules` to route `t_1` to `t_2` when update config, so create table `t_2` here